packages/sdk/src/interfaces/types.ts

```diff
@@ -48,6 +48,13 @@ export enum MessageStatus {
    */
   UNCONFIRMED_L1_TO_L2_MESSAGE,
 
+  /**
+   * Message is an L1 to L2 message and the transaction to execute the message failed.
+   * When this status is returned, you will need to resend the L1 to L2 message, probably with a
+   * higher gas limit.
+   */
+  FAILED_L1_TO_L2_MESSAGE,
```

Comment:
Looks good, this happens similarly to the relay case right? When a user defines their own gas limit for depositing/withdrawing?

Reply:
Yeah, exactly. If the user supplies too little gas the relay could fail. Could also fail if the contract being triggered reverts for whatever reason.